src/@types/navigation/Owner.ts

```diff
@@ -13,12 +10,7 @@ export type OwnerTabParamList = {
   Reservation: undefined;
 };
 
-export type OwnerStackScreenProps<T extends keyof OwnerStackParamList> = CompositeScreenProps<
-  StackScreenProps<OwnerStackParamList, T>,
-  AuthStackScreenProps<keyof AuthStackParamList>
->;
-
-export type OwnerTabScreenProps<T extends keyof OwnerTabParamList> = CompositeScreenProps<
-  BottomTabScreenProps<OwnerTabParamList, T>,
-  OwnerStackScreenProps<keyof OwnerStackParamList>
+export type OwnerTabScreenProps<T extends keyof OwnerTabParamList> = BottomTabScreenProps<
+  OwnerTabParamList,
+  T
```

Comment:
I believe the Tab is part of the Stack in Owner. So to parse the OwnerTabScreenProps you must compose it with the OwnerStackScreenProps, so the tab screens is able to navigate through the owner stack, as well as shift through tabs.

```js
export type OwnerStackScreenProps<T extends keyof OwnerStackParamList> = StackScreenProps<OwnerStackParamList, T>;

export type OwnerTabScreenProps<T extends keyof OwnerTabParamList> = CompositeScreenProps<BottomTabScreenProps<OwnerTabParamList, T>, OwnerStackScreenProps<keyof OwnerStackParamList>>;
```